### PR TITLE
Review mockups

### PR DESCRIPTION
**Pull request by calvinlu3**
This fixes https://github.com/oncokb/oncokb/issues/3698

### CODE REVIEW

File: src/main/webapp/app/pages/curation/collapsible/Collapsible.tsx [hunk 1]
```diff
@@ -83,14 +85,23 @@ export default function Collapsible({
         )}
         ref={node => {
           if (node && borderLeftColor) {
+            const hadBackgroundColor = backgroundColor;
             if (disableCollapsible) {
               borderLeftColor = DISABLED_NEST_LEVEL_COLOR;
+              if (!hadBackgroundColor) {
+                backgroundColor = DISABLED_NEST_LEVEL_COLOR;
```
Comment (bprize15) [inline src/main/webapp/app/pages/curation/collapsible/Collapsible.tsx:92]: Do we need `hadBackgroundColor`? Can we just change to `if (!backgroundColor)`?

File: src/main/webapp/app/shared/util/firebase/firebase-review-utils.ts [hunk 2]
```diff
@@ -352,6 +355,9 @@ const findMutationEffectReviews = (
         { [fieldKey]: mutationEffect[reviewKey].lastReviewed },
         isNestedUnderCreateOrDelete(parentReview)
       );
+      if (fieldKey === 'oncogenic' || fieldKey === 'effect') {
```
Comment (bprize15) [inline src/main/webapp/app/shared/util/firebase/firebase-review-utils.ts:358]: Solution works for now, but will break with new listener changes. Can leave as is, but might want to consider a fix that checks the input type or some way that will not break (if that is possible).

File: src/main/webapp/app/pages/curation/collapsible/Collapsible.tsx (hunk 1 above)
Comment (calvinlu3) [reply src/main/webapp/app/pages/curation/collapsible/Collapsible.tsx:92]: Mainly used in:
```
getHexColorWithAlpha(backgroundColor, hadBackgroundColor ? 0 : 0.05)
```
By default, the background color = border left color with 0.05 opacity. But if background color was passed as prop, then it should not have opacity.

File: src/main/webapp/app/shared/util/firebase/firebase-review-utils.ts (hunk 2 above)
Comment (calvinlu3) [reply src/main/webapp/app/shared/util/firebase/firebase-review-utils.ts:358]: Yeah this is true. Don't know if there's a good way to know if a field is a checkbox/radio without probably hard-coding. I will come back and fix it

**Review by bprize15 — APPROVED** [review]

**Pull request merged**